src/authenticators/direct_authenticator/mod.rs

```diff
@@ -12,11 +12,12 @@
 // WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-//! The direct authenticator
+//! Direct authenticator
 //!
-//! The `DirectAuthenticator` will return the authentication value parsed as a UTF-8 string.
-//! This authentication method has no security value and does not check for the integrity of the
-//! clients' requests.
+//! The `DirectAuthenticator` implements the [direct authentication](https://parallaxsecond.github.io/parsec-book/parsec_service/system_architecture.html#authentication-tokens)
+//! functionality set out in the system architecture. As such, it attempts to parse the request
+//! authentication field into an UTF-8 string and returns the result as an application name.
+//! This authenticator does not offer any security value and should only be used for testing.
```

Comment:
> and should only be used for testing.

It can be used securely in a context where an Identity Provider is not needed, where all the clients that are currently using Parsec trust each other not to be malicious and not to use each other application name.

Reply:
Indeed, wasn't sure what to say there. Will change to something closer to that

Reply:
Done